rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+scheduled election, the Secretary must call for a special election for the vacant seats.  The election process is the
+same as for any other election, but the newly elected TC members will serve only until the next regular election.
+
+Should the number of TC members fall to zero, the senior-most Release Manager of the latest stable release will step
+ in to act as TC Secretary and immediately call for a new vacancy election.
+
+==== Work flow ====
+
+The TC is primarily a reactive body, and is not expected to proactively search PRs for issues.  However, TC members
+are permitted to comment on any open issue, and contributors are encouraged to give their comments appropriate weight.
+
+Should a dispute or question about a change arise, the TC may be called on to resolve it.
+
+  - Any core developer (including a TC member) may ask for TC resolution of an issue by adding the label `tc-hold` to it and @-mentioning the TC GitHub team.
+  - An issue with the `tc-hold` tag MUST NOT be merged.
+  - The TC members will confer in whatever manner they deem most convenient, public or private or both.
+    * The TC SHOULD consult with relevant other individuals, including the core developers involved in the issue, Release
+      Managers, and impacted extension maintainers as appropriate.
+  - The TC MUST deliver a decision within one month, via the mechanism described below.
+  - The TC Secretary MUST announce the decision and reasoning for it on the issue in question and remove the `tc-hold` tag.
+  - The decision of the TC is binding.  That means a relevant change MUST NOT be merged until and unless it conforms with the TC's decision.  Depending on the decision, the core developer involved may merge the issue, close it, or revise it accordingly.
+
+If an issue has been marked `tc-hold` for more than two weeks with no response or acknowledgement from the TC, the
+developer SHOULD post to the Internals mailing list a notification of absence.
+
+If a notification of absence receives no response from the TC or any member of it for one month, then the entire TC
+is considered AWOL and removed.  New elections must then be immediately called as described above.
+
+The election of a new TC will "reset the clock" on any outstanding `tc-hold` issues.
+
+==== Decision process ====
+
+=== User-facing changes ===
+
+The RFC process will continue to be used for user facing changes.  However, it will no longer be used for
+non-user-facing technical decisions.
+
+TC members have no special role or authority in the RFC process.
+
+=== Process changes ===
+
+The RFC process will continue to be used for process changes, including changes to the role of the TC.
+
+=== Technical changes ===
+
+As noted, the TC has no special role or authority on user-facing changes.  The TC may, however, decide on the
+implementation of those features. In other words the TC can decide about implementation and ABI but not about API.
```

Comment:
The term API is ambiguous in this context. It can decide on the C API, but not PHP API that was voted on.

Reply:
Fixed, thanks.